sucuri-cleanup.py

```diff
@@ -1,22 +1,115 @@
-# So far I'm mainly trying to collect the bits of code that I'm going to need for the core functionality of the script
-# this is by no means functional. And so far everything pretty much looks like a global. I'll work these bits down
-# into functions as I go, at least that's the plan so far ;)
+#ITS ALIVE!!!!!!!
+#And now, it's also more optimized. We've removed the nested for looping, and made some other minor enhancements.
+#The script now also checks to see if it's being run from an install's folder, and if not, it fails out with debug
+#Command line options for debug output and executing from a local command prompt have been added to streamline future additions to the code
+#did some cleanup to make the code prettier, and hopefully easier to read.
 
 import sys
 import os
 import requests
 import re
-
-salts = requests.get('https://api.wordpress.org/secret-key/1.1/salt/')
+import _mysql as dbcon
+debug = False
+local = False
```

Comment:
Consider making these two variables (debug and local) options that can be input via the command line. There's a great python library to help with it too! It's called `argparse` -- check it out: https://docs.python.org/3.3/library/argparse.html